docs/host/host.md

```diff
@@ -80,3 +80,81 @@ If you are testing Harvester in a QEMU environment, you'll need to use QEMU v6.0
 
 ![Edit Config](/img/v1.1/host/edit-config.png)
 ![Add Disks](/img/v1.1/host/add-disks.png)
+
+## Ksmtuned Mode
+
+Ksmtuned is a KSM automation tool deployed as a DaemonSet to run Ksmtuned per node and start or stop KSM by setting the available memory percentage ratio (**Threshold Coefficient**), By default, you need to manually enable Ksmtuned on each node UI. You can see the KSM statistics from the node UI ([see the KSM in detail](https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html#ksm-daemon-sysfs-interface))
```

Comment:
```suggestion
Ksmtuned is a KSM automation tool deployed as a DaemonSet to run Ksmtuned on each node. It can start or stop KSM by setting the available memory percentage ratio (**Threshold Coefficient**), By default, you need to manually enable Ksmtuned on each node UI. You can see the KSM statistics from the node UI (for details, see [KSM](https://www.kernel.org/doc/html/latest/admin-guide/mm/ksm.html#ksm-daemon-sysfs-interface)).
```